package.json

```diff
@@ -3,8 +3,10 @@
   "version": "1.3.1",
   "description": "Common Brightspace eslint configs.",
   "main": "index.js",
```

Comment:
A nice new feature we can leverage here is to ditch `main` and use [package.json exports](https://nodejs.org/api/packages.html#exports-sugar). Since we only have the 1, I think we could just go:

```json
{
  "exports": "./index.js"
} 
```

This way we lock down what consumers can import -- i.e. they couldn't import from `utils.js` or `lit-config.js` directly.

Reply:
I included some things in `utils.js` that would help simplify config files and handle file overrides properly, so I included the file too.